ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/Util/Time.hs

```diff
@@ -24,40 +19,15 @@ import           GHC.Generics (Generic)
 import           GHC.Stack
 
 import           Cardano.Prelude (NoUnexpectedThunks)
-
-import           Ouroboros.Network.Block (SlotNo (..))
+import           Cardano.Slotting.Slot
 
 import           Ouroboros.Consensus.BlockchainTime
 import           Ouroboros.Consensus.Util.IOLike
 import           Ouroboros.Consensus.Util.ResourceRegistry
 import           Ouroboros.Consensus.Util.STM
 
 {-------------------------------------------------------------------------------
-  Fixed time
--------------------------------------------------------------------------------}
-
--- | A 'BlockchainTime' that is fixed to the given slot.
---
--- 'onSlotChange_' does nothing.
-fixedBlockchainTime :: MonadSTM m => SlotNo -> BlockchainTime m
-fixedBlockchainTime slot = BlockchainTime {
-      getCurrentSlot = return slot
-    }
-
-{-------------------------------------------------------------------------------
-  Settable time
--------------------------------------------------------------------------------}
-
--- | The current slot can be changed by modifying the given 'StrictTVar'.
---
--- 'onSlotChange_' is not implemented and will return an 'error'.
-settableBlockchainTime :: MonadSTM m => StrictTVar m SlotNo -> BlockchainTime m
-settableBlockchainTime varCurSlot = BlockchainTime {
-      getCurrentSlot = readTVar varCurSlot
-    }
```

Comment:
I was expecting these to move along to `Test.Util.Time` instead of moving to `BlockchainTime.API`, but since they're going away soon anyway, it doesn't really matter.

Reply:
Yeah, I considered that, but they return a `BlockchainTime` not a `TestBockchainTime` and so that didn't feel quite right to me. Add yes, they are gone in https://github.com/input-output-hk/ouroboros-network/pull/1989 .